components/SubHeaderButtons.js

```diff
@@ -1,44 +1,79 @@
 import React, { useState } from 'react';
-import { StyleSheet, View, Text, TouchableOpacity } from "react-native";
+import { StyleSheet, View, Text, TouchableOpacity, Button } from "react-native";
+import { Picker, Icon } from "native-base"
 
 export default function SubHeaderButtons(props) {
 
-    const [text, setText] = useState(false)
-    togglePicker = () => {
-        setText(!text)
-    }
-
+    const [ShowPicker, setShowPicker] = useState(true)
+  const [category, setCategory] = useState('')
+  const pick = () => {
+    setShowPicker(true)
+  }
     return (
         <View style={styles.main}>
             <View style={{ flex: 1, backgroundColor: 'blue' }}>
                 <TouchableOpacity style={styles.btn}
                     onPress={props.changeMode}
                 >
-                    <Text style={{ fontSize: 17, color: 'white' }}>{props.mode}</Text>
+                    <Text style={{ fontSize: 19, color: 'white' }}>{props.mode}</Text>
```

Comment:
pls add a condition for the modes
( if props.mode = explore -> EXPLORE
 if props.mode = play -> PLAY
)